seqr/views/apis/report_api_tests.py

```diff
@@ -209,30 +208,30 @@ class ReportAPITest(AuthenticationTestCase):
 
     def test_seqr_stats(self):
         url = reverse(seqr_stats)
-        self.check_staff_login(url)
+        self.check_analyst_login(url)
```

Comment:
"check_analyst_login" checks that non-analysts can't access it, thats the entire point of that function. I don't know what on earth else we could use as " a non-analyst user test" because non-analyst users don't have access, which is what we already tested